docs/source/documentation/concepts.rst

```diff
@@ -160,6 +160,31 @@ There are two ways to run several tasks in parallel on a same organization with
 .. TODO:: Explain what is substra tools
 
 
+Substra channels
+----------------
+
+A channel is a group of Substra `organizations <https://docs.substra.org/en/stable/additional/glossary.html#term-Organization>`_ which operates on a common set of assets.
+
+Having multiple channels is needed if you want to have multiple independent projects on the same network. 
+
+In the scheme below, we represent a Substra network with two channels:
+
+- Channel A with the three organizations
+- Channel B with only two organizations (A and B)
+
+.. image:: /static/schemes/channel_schema.png
+
+Users only belong to one channel
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A user can only belong to one channel. A channel has its own users, its own datasets, compute plans, tasks and functions. A user from another channel has no visibility on this. A user has no visibility on a channel they are not a part of.
```

Comment:
It 's a bit unclear to me what "this" is in the second sentence. Does this reflect your intent?

```suggestion
A user can only belong to one channel. A channel has its own users, its own datasets, compute plans, tasks and functions -- all invisible to users from other channels. A user has no visibility on a channel they are not a part of.
```

I'm not sure I understand what the third sentence is trying to say:
 - if it restates the previous sentence, it might be superfluous
 - if it's saying users can't even know about the existence of channels they don't belong to, I think it could be spelled out more explicitely.